backend/api/tests/views/test_views_performance.py

```diff
@@ -51,7 +55,14 @@ def setUp(self):
             )
             for i in range(3)
         ]
-        self.performances = [factory.create_performance(self.compute_tasks[i], self.metric) for i in range(3)]
+        self.performances = [
+            factory.create_performance(
+                self.compute_tasks[i],
+                self.metric,
+                identifier="performance",
```

Comment:
Should we use const for the `performance` string ?

Reply:
I am not sure because identifiers are free text values